website/docs/r/service_principal.html.markdown

```diff
@@ -36,43 +36,47 @@ resource "azuread_service_principal" "example" {
 
 The following arguments are supported:
 
-* `application_id` - (Required) The ID of the Azure AD Application for which to create a Service Principal.
+- `application_id` - (Required) The ID of the Azure AD Application for which to create a Service Principal.
```

Comment:
Could we not change these from `*` to `-` for consistency with the rest of the provider docs?